README.md

```diff
@@ -3,27 +3,17 @@
 [![Build Status](https://ci.jenkins.io/buildStatus/icon?job=Plugins%2Fidea-stapler-plugin%2Fmaster)](https://ci.jenkins.io/job/Plugins/job/idea-stapler-plugin/job/master/)
 [![Join the chat at https://gitter.im/jenkinsci/idea-stapler-plugin](https://badges.gitter.im/jenkinsci/idea-stapler-plugin.svg)](https://gitter.im/jenkinsci/idea-stapler-plugin)
 
-[//]: # (Content between "Plugin description" markers are extracted by gradle build. No markdown formatting. Simple html only.)
-<!-- Plugin description -->
-<p>Assists with development of Jenkins core and plugins.</p>
-<p>Supports following technologies that are not covered by any other IntelliJ plugins:</p>
-<ul>
-  <li><a href="https://stapler.kohsuke.org/">Stapler HTTP request handling engine</a></li>
-  <li><a href="https://commons.apache.org/proper/commons-jelly/index.html">Jelly : Executable XML</a></li>
-  <li><a href="https://commons.apache.org/proper/commons-jexl/">Java Expression Language (JEXL)</a></li>
-</ul>
-<!-- Plugin description end -->
+[//]: # (Update plugin features in build.gradle as well if they are worth mentioning in key features)
```

Comment:
It was nice having the description here but it meant that the description on the marketplace was quite average as it just included a brief mention of the technologies supported and not the features.

We could convert all of the features section to HTML as well and just include all of them but in the build.gradle I selected the features that I thought were most worth mentioning...